src/plugins/field_formats/common/converters/number.test.ts

```diff
@@ -27,4 +27,12 @@ describe('NumberFormat', () => {
 
     expect(formatter.convert('12.345678')).toBe('12');
   });
+
+  test('object input', () => {
+    const formatter = new NumberFormat({}, getConfig);
+
+    expect(
+      formatter.convert({ min: 150, max: 1000, sum: 5000, value_count: 10 })
+    ).toMatchInlineSnapshot(`"{\\"min\\":150,\\"max\\":1000,\\"sum\\":5000,\\"value_count\\":10}"`);
+  });
```

Comment:
one last ask, let's cover the html part 
```suggestion
  test('object input', () => {
    const formatter = new NumberFormat({}, getConfig);

    expect(
      formatter.convert({ min: 150, max: 1000, sum: 5000, value_count: 10 })
    ).toMatchInlineSnapshot(`"{\\"min\\":150,\\"max\\":1000,\\"sum\\":5000,\\"value_count\\":10}"`);
  });

    expect(formatter.convert({ min: 150, max: 1000, sum: 5000, value_count: 10 }, 'html'))
      .toMatchInlineSnapshot(`
      "{
        \\"min\\": 150,
        \\"max\\": 1000,
        \\"sum\\": 5000,
        \\"value_count\\": 10
      }"
    `);    
```